### ISSUE TITLE
How to test dev changes

### ISSUE THREAD

**Issue by saitodisse** [cerebral/cerebral-module-firebase#4]
Hello @christianalfoni, this module is very nice and simple. I want to contribute. I believe there is error on `listenTo` helper. It is not clearing listeners when is called for the second time. 

My question is if there is any clever way that you do to make changes on this module?

I keep getting this error when I try to test this module from my app:

``` js
// from app
import FirebaseModule from '../../cerebral-module-firebase/dist/index.js';

// ERROR
Uncaught Error: Cannot find module "firebase"
```

> Now I understand why other modules are not written in ES6 with webpack

@garth use babel only on this package https://github.com/cerebral/cerebral-computed-fuse. Maybe this is the way to go.

As it expects an external `firebase` I can't test because receives `firebase` as external lib on webpack: https://github.com/cerebral/cerebral-module-firebase/blob/master/webpack.build.config.js#L27-L29

**Comment by saitodisse** [comment]
I workaround the problem and make this works. I don't know if will break other things but what I did was to remove all firebase imports and externals configuration from webpack. Seen's to work very well. 
https://github.com/saitodisse/cerebral-module-firebase/commit/5872847983d3f3e49c2d40568ca07e4b2e522911

**Comment by christianalfoni** [comment]
Yeah, it is supposed to have build step here :) Let me add that now, doing a couple of other changes as well.

Let me check the `listenTo` thingy as well, you are perfectly right. Same event, path and signal should not be added twice :)

**Comment by christianalfoni** [comment]
Hm, looking at the code `listenTo` does turn off event listener on existing event, path and signal registered. You have an example where it goes wrong?

**Comment by christianalfoni** [comment]
Ah, yeah... I see the project does create a dist, hehe. So this should work :)

**Comment by saitodisse** [comment]
@christianalfoni What do you think about remove all firebase imports like I did? https://github.com/cerebral/cerebral-module-firebase/issues/4#issuecomment-244265589
